qtl.metadata.R

```diff
@@ -108,7 +108,7 @@ if (analysis == "eQTL") {
     scores.sd <- data.table(matrix.colname=names(scores.sd), sdscore=as.numeric(scores.sd))
     setkey(scores.sd, matrix.colname)
```

Comment:
I think lines with setkey are no longer needed. Can you remove them here and also for 'pQTL' and see if everything runs correctly?

Reply:
Yeah, running them today with `on` seems to work fine today, so I have left the setkey.